src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
```

Comment:
Javadocs for the class too will be good to have!

Reply:
Agree with @lwj1711